src/modules/common/renderHtml.js

```diff
@@ -6,6 +6,36 @@ import type { ThemeType } from '../theme/constants/theme'
 
 // language=JavaScript
 const renderJS = (files: PageResourceCacheStateType) => `
+// Catching occurring errors
+(function() {
+  function reportError (message) {
+    if (!window.ReactNativeWebView) {
+      return window.setTimeout(function() { reportError(message) }, 100)
+    }
+
+    window.ReactNativeWebView.postMessage(JSON.stringify({ type: "error", message: message }))
+  };
+  
+  window.onerror = function(msg, url, lineNo, columnNo, error) {
+    // from https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onerror
+    var string = msg.toLowerCase()
+    var substring = "script error"
```

Comment:
Weird naming, also `string`

Reply:
Since this is the referenced code from mozilla, I'd probably leave it like this.